doc/whats_new.rst

```diff
@@ -20,6 +20,7 @@ BUG
 API
 ~~~
 
+    - Added logscale epoch plotting option via `logscale=True` in :func:`mne.viz.plot_epochs_image`, by `Sagun Pai`_
```

Comment:
can you try something using the sample data stim channel without hacking
the data?